File: Q5.cs
using System;

namespace ConstructorExample
{
    class Person
    {
        public string Name { get; set; }
        public int Age { get; set; }

        // Default constructor
        public Person()
        {
            Name = "Unknown";
            Age = 0;
            Console.WriteLine("Default Constructor called.");
        }

        // Parameterized constructor
        public Person(string name, int age)
        {
            Name = name;
            Age = age;
            Console.WriteLine("Parameterized Constructor called.");
        }

        public void DisplayInfo()
        {
            Console.WriteLine($"Name: {Name}, Age: {Age}");
        }
    }

    class Program
    {
        static void Main(string[] args)
        {
            // Using default constructor
            Person person1 = new Person();
            person1.DisplayInfo();

            // Using parameterized constructor
            Person person2 = new Person("Alice", 30);
            person2.DisplayInfo();
        }
    }
}

File: Q5.vb
Module Module1

    Class Person
        ' Properties
        Public Property Name As String
        Public Property Age As Integer

        ' Default Constructor
        Public Sub New()
            Name = "Unknown"
            Age = 0
        End Sub

        ' Parameterized Constructor
        Public Sub New(name As String, age As Integer)
            Me.Name = name
            Me.Age = age
        End Sub

        ' Method to display person's information
        Public Sub DisplayInfo()
            Console.WriteLine($"Name: {Name}, Age: {Age}")
        End Sub
    End Class

    Sub Main()
        ' Using the default constructor
        Dim person1 As New Person()
        person1.DisplayInfo()

        ' Using the parameterized constructor
        Dim person2 As New Person("Alice", 30)
        person2.DisplayInfo()

        Console.ReadLine()
    End Sub

End Module
